### PR TITLE
Task/#850 remove object level permission required mixin

### PR DESCRIPTION
**Pull request by derrix060**

### CODE REVIEW

**Comment by derrix060** [comment]
I thought that the job would be straightforward, but looks like is not only replace. I tested using the docker composer and I got the same results, but I didn't believe that I was executing the right command haha.

I will try to investigate the issues on Friday.

**Comment by LegoStormtroopr** [comment]
Strange that is happening.

The tests and permissions shouldn't need to be changed, I think its just isolated to how discussions are being looked up.

**Comment by derrix060** [comment]
Its missing [one detail](https://github.com/brack3t/django-braces/pull/229/files#diff-0890f4a5aaf36e6f7e6de147b430cf61R72): 

> If you need to enforce permissions against a given object, set the ``object_level_permissions`` attribute to ``True``.

I don't know when is not required to enforce permissions, but I have to set this as True on the classes from discussions.py

ps: sorry for the delay, I'm travelling on Friday, so I was preparing the trip

File: python/aristotle-metadata-registry/aristotle_mdr/views/workgroups.py [hunk 1]
```diff
@@ -47,7 +46,7 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class WorkgroupView(LoginRequiredMixin, WorkgroupContextMixin, ObjectLevelPermissionRequiredMixin, DetailView):
+class WorkgroupView(LoginRequiredMixin, WorkgroupContextMixin, PermissionRequiredMixin, DetailView):
```
Comment (LegoStormtroopr) [inline python/aristotle-metadata-registry/aristotle_mdr/views/workgroups.py]: Note to self: WorkgroupContent mixin has `object_level_permissions = True`.
So this change is approved.

File: python/aristotle-metadata-registry/aristotle_mdr/views/registrationauthority.py [hunk 2]
```diff
@@ -58,7 +57,7 @@ def get_success_url(self):
         return reverse('aristotle:registrationauthority_details', kwargs={'iid': self.object.id})
 
 
-class AddUser(LoginRequiredMixin, ObjectLevelPermissionRequiredMixin, UpdateView):
+class AddUser(LoginRequiredMixin, PermissionRequiredMixin, UpdateView):
```
Comment (LegoStormtroopr) [inline python/aristotle-metadata-registry/aristotle_mdr/views/registrationauthority.py]: requires `object_level_permissions = True`

File: python/aristotle-metadata-registry/aristotle_mdr/views/registrationauthority.py [hunk 3]
```diff
@@ -140,7 +139,7 @@ class MembersRegistrationAuthority(LoginRequiredMixin, PermissionRequiredMixin,
     context_object_name = "item"
 
 
-class EditRegistrationAuthority(LoginRequiredMixin, ObjectLevelPermissionRequiredMixin, UpdateView):
+class EditRegistrationAuthority(LoginRequiredMixin, PermissionRequiredMixin, UpdateView):
```
Comment (LegoStormtroopr) [inline python/aristotle-metadata-registry/aristotle_mdr/views/registrationauthority.py]: requires `object_level_permissions = True`

**Pull request closed without merging**